### PR TITLE
Temporary fixes for CheckstyleBear & RamllintBear

### PR DESCRIPTION
**Pull request by jayvdb**
Workarounds for https://github.com/coala/coala-bears/issues/1461 and https://github.com/coala/coala-bears/issues/1470 .

### CODE REVIEW

File: bears/java/CheckstyleBear.py [hunk 1]
```diff
@@ -53,9 +53,9 @@ class CheckstyleBear:
 
     def setup_dependencies(self):
         type(self).checkstyle_jar_file = self.download_cached_file(
-            'http://sourceforge.net/projects/checkstyle/files/checkstyle/6.15'
-            '/checkstyle-6.15-all.jar',
-            'checkstyle.jar')
+            'https://github.com/Berkeley-CS61B/skeleton-sp16/raw/master'
```
Comment (adtac) [inline bears/java/CheckstyleBear.py]: can we use our own fork of that? we don't know when that link would 404

Comment (jayvdb) [inline bears/java/CheckstyleBear.py]: Or create a new repo and put all of our fetched files in it?

File: package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
     "happiness": "~7.1.2",
     "jshint": "~2",
     "postcss-cli": "~2",
-    "ramllint": "~1.2.2",
+    "ramllint": ">1.2.2 <1.2.4",
```
Comment (adtac) [inline package.json]: Clarification: should this be `>=1.2.2` or just `>1.2.2`?

**Comment by adtac** [comment]
ack https://github.com/coala/coala-bears/pull/1471/commits/fd305e0088c0d1b5a83ab8e0efd3e8ebaae25e69

File: package.json (hunk 2 above)
Comment (jayvdb) [inline package.json]: Oh, well spotted. Yes, 1.2.2 is still ok.

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**